dvc/tree/base.py

```diff
@@ -48,7 +48,7 @@ class BaseTree:
     scheme = "base"
     REQUIRES = {}
     PATH_CLS = URLInfo
-    JOBS = 4 * cpu_count()
+    JOBS = 4
```

Comment:
This is misleading. And also seems incorrect, as there is no way to have `jobs = 1` in the config, as it will now always multiply it by NCPU in `jobs_count` below.

Reply:
Ok, got it, is the value must be independent of CPU count?

Reply:
Yes, same as it was before.